api/internal/utils/pid.go

```diff
@@ -26,6 +26,9 @@ import (
 
 // WritePID write pid to the given file path.
 func WritePID(filepath string) error {
+	if _, err := os.Stat(filepath); err == nil {
+		return fmt.Errorf("instance of Manager API already running: a pid file exists in %s", filepath)
```

Comment:
Does this mean the manager can't start successfully if there exists a PID file?

Reply:
We may add an option to start forcely.

Reply:
Hii guys, as we are now supporting manager-api as an os service, the PID file should not be overwritten unknowingly, because then `./manager-api stop` won't work. we have to take help from the os service manager explicitly.

> We may add an option to start forcely.
Yes, we can do that.